thirdparty/3DConnexion/README.txt

```diff
@@ -0,0 +1,3 @@
+This directory contains part of the the 3Dconnexion 3DxWare_SDK v4
+The full sdk can be obtained from https://3dconnexion.com/software-developer-program/
```

Comment:
Is there a good way to keep this up-to-date (e.g., when v5 comes out?)

Reply:
No, as we are shipping some SDK's headers with the solution they will need to be updated manually when a new release comes out.